pkg/assembler/backends/inmem/hasSBOM.go

```diff
@@ -27,13 +27,12 @@ import (
 
 type hasSBOMList []*hasSBOMStruct
 type hasSBOMStruct struct {
-	id         uint32
-	pkg        uint32
-	src        uint32
-	uri        string
-	annotation string
-	origin     string
-	collector  string
+	id        uint32
+	pkg       uint32
+	src       uint32
+	uri       string
+	origin    string
+	collector string
```

Comment:
We're not storing the new fields?

Reply:
this was an old implementation where I added an `annotation` field. So I removed that for now and will create another PR to update the backends. I wanted to get feedback on the schema before moving forward.

Reply:
Just added to this PR.